src/package/nuspec/Microsoft.TestPlatform.Portable.nuspec

```diff
@@ -8,7 +8,7 @@
     <owners>Microsoft</owners>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
     <description>The cross platform Microsoft Test Platform.</description>
```

Comment:
Same description for both `Microsoft.TestPlatform.Portable` and `Microsoft.TestPlatform`

Reply:
@pvlakshm Please suggest description for these packages.